worker/httpserver/cert_test.go

```diff
@@ -37,6 +37,7 @@ func (s *certSuite) SetUpTest(c *gc.C) {
 		"https://0.1.2.3/no-autocert-here",
 		nil,
 		testSNIGetter(coretesting.ServerTLSCert),
+		loggo.Logger{},
```

Comment:
Personally I generally use loggo.GetLogger("test"), but this will work, and will write things at the `<root>` level.

Reply:
Will update